### ISSUE TITLE
Specify OpenTofu version in standalone mode

### ISSUE THREAD

**Issue by timofurrer** [opentofu/get.opentofu.org#13]
When running:

```sh
./install-opentofu.sh --install-method standalone --opentofu-version 1.6.0
```

the script currently fails with:

```
Error: Unknown option: --opentofu-version.
```

Judging by looking at the script it looks like the `--opentofu-version` option is simply not implemented.

This is currently a blocker for us and it would be great to have a fix for this :)

**Comment by timofurrer** [comment]
I contributed a minimal patch at https://github.com/opentofu/get.opentofu.org/pull/14+

**Comment by janosdebugs** [comment]
Closing this as it is fixed in #14 :) Thank you again!
